mkosi/__init__.py

```diff
@@ -2680,7 +2705,9 @@ def install_arch(args: MkosiArgs, root: Path, do_run_build_script: bool) -> None
     if args.release is not None:
         MkosiPrinter.info("Distribution release specification is not supported for Arch Linux, ignoring.")
 
-    if args.mirror:
+    if args.use_mirror_verbatim and args.mirror:
```

Comment:
Shouldn't this be moved inside the original `if args.mirror` conditional?

Reply:
this is rearranged again by a later commit, so no longer applies